BeefLibs/corlib/src/System.bf

```diff
@@ -178,6 +178,18 @@ static
 		}
 	}
 
+	public static mixin DeleteDictionaryAndValues(var container)
+	{
+		if (container != null)
+		{
+			for (var value in container)
+			{
+				delete value.value;
+			}
```

Comment:
Wouldn't it be more appropriate to call the loop variable an `item`?

Ex:
```c#
for (var item in container)
{
    delete item.value;
}
```

Reply:
Or `element`. I often use `kv` for key/value pairs.